api/api-doc/column_family.json

```diff
@@ -84,6 +84,14 @@
                      "type":"string",
                      "paramType":"path"
                   },
+                  {
+                     "name":"sf",
```

Comment:
please use a non-abbreviated name

Reply:
I chose it by following the snapshot api example so to stick to the existing convention.
See https://opensource.docs.scylladb.com/stable/operating-scylla/nodetool-commands/snapshot.html
I think that the readability is more important in the tool level and less at the api level that is less interactive.
Ideally, we could support both short and long aliases for the options also at the api parsing level.

Reply:
In the next version I kept the api option as `cf` is in next version for consistency reasons with the snapshot api.
scylla-nodetool will take only the long option name: `--skip-flush`

Reply:
Ok.

Reply:
I don't understand the consistency argument. Someone picked a bad name for a related CLI all, so we must repeat it?

Also, let's avoid the negative and call it flush-memtables and default it to true.

Reply:
ok, I'l change the name and use positive semantics.